javaScript.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+  <body>
+    <p id="demo">Hi</p>
+    <script>
+      demo.getElementById("demo").innerHTML="This is a cool Java script stuff";
```

Comment:
did you try this? did it work?
I believe you need to declare `demo` first before you can use. something like the code below:
```
var demo = document.getElementById('demo');
demo.innerHTML  = 'This is a cool Java script stuff';
```